### ISSUE TITLE
[table-groups] False-positive with a nested component that has a tagName

### ISSUE THREAD

**Issue by boris-petrov** [ember-template-lint/ember-template-lint#1656]
I see that a [similar](https://github.com/ember-template-lint/ember-template-lint/blob/v2.15.0/test/unit/rules/table-groups-test.js#L102) check is done in the tests, however I have another case:

```hbs
<table>
  {{#some-glimmer-component-that-has-no-hbs as |foobar|}}
    {{some-component foobar=foobar tagName="tbody"}}
  {{/some-glimmer-component-that-has-no-hbs}}
</table>
```

As you can see, before the component with `tagName` I have another component that doesn't have any HBS. This is a false-positive, however, I understand that this is a bit too much so I won't be offended if you close the issue as a "won't fix". :smile:

**Comment by MelSumner** [comment]
I think you can put an inline exception here, I don't think we can reasonably determine that a component has no hbs. Will close as a won't fix for now, but please re-open if you have an idea for how to approach the problem. Thank you!
